article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -58,7 +60,12 @@ case class PageData(
     edition: String,
     contentType: Option[String],
     commissioningDesks: Option[String],
-    subMetaLinks: SubMetaLinks
+    subMetaLinks: SubMetaLinks,
+
+    // AMP specific
```

Comment:
These will probably be not AMP specific for long, but I wanted to flag it just in case they end up only used by AMP and this represents a genuine divergence of requirements.